### PR TITLE
feature/vxd-23: aurora stack

### PR DESCRIPTION
**Pull request by gdenn**

### CODE REVIEW

**Review by mastercam123 — COMMENTED** [review]
all looks good for me

**Comment by gdenn** [comment]
@mastercam123 you need to push the approval button, otherwise I can't merge ^^

**Review by mastercam123 — APPROVED** [review]
lokks all fine

File: cft/aurora.yml [hunk 1]
```diff
@@ -40,16 +43,15 @@ Resources:
...
-      MultiAZ: true
+      DBInstanceIdentifier: !Sub '${DBInstanceName}-instance1'
+      DBSubnetGroupName: !Ref AuroraSubnetGroup
       Engine: aurora-postgresql
       DBClusterIdentifier: !Ref AuroraDBCluster
-      PubliclyAccessible: false
-      DBInstanceClass: db.t2.micro
\ No newline at end of file
+      PubliclyAccessible: 'false'
+      DBInstanceClass: db.t3.medium
```
[the remark this answers is not in the document]
Comment (gdenn) [reply cft/aurora.yml:57]: that makes sense, i'll add a parameter schwiftly

[the remark this answers is not in the document]
Comment (gdenn) [reply cft/aurora.yml]: adjusted

**Pull request merged**